packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -277,6 +280,11 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     }
   };
 
+  private _calculatePrecision = (props: ISpinButtonProps & DefaultProps) => {
```

Comment:
non-blocking suggestion, but you can consider aliasing `ISpinButtonProps & DefaultProps` to something like `ISpinButtonInternalProps` so you don't have to type `ISpinButton & DefaultProps` over and over again